### DISCUSSION TITLE
how to change the order of each section?

### DISCUSSION THREAD

**Discussion by zeyunlu** [alshedivat/al-folio#349] [Q&A] [answered]
Seems the default "About", "Blogs", "Projects", "Publications" are in alphabet order, how do I make it such as "Blogs", "About", "Nancy", "Information" ?

Thanks.

**Comment by mbarbetti** [comment, marked as answer]
The order of the sections in the navbar is set in [header.html](https://github.com/alshedivat/al-folio/blob/master/_includes/header.html).
Actually the first section is **about** (see from [line 25](https://github.com/alshedivat/al-folio/blob/master/_includes/header.html#L25)):
```HTML
<!-- About -->
<li class="nav-item {% if page.title == 'about' %}active{% endif %}">
  <a class="nav-link" href="{{ '/' | relative_url }}">
    about
    {% if page.title == "about" %}
    <span class="sr-only">(current)</span>
    {% endif %}
  </a>
</li>
```
Then **blog** (see from [line 34](https://github.com/alshedivat/al-folio/blob/master/_includes/header.html#L34)):
```HTML
{% if site.blog_name %}
<!-- Blog -->
<li class="nav-item {% if page.url contains 'blog' %}active{% endif %}">
  <a class="nav-link" href="{{ '/blog/' | relative_url }}">
    blog
    {% if page.title == "blog" %}
    <span class="sr-only">(current)</span>
    {% endif %}
  </a>
</li>
{% endif %}
```
And finally the **rest of pages** in alphabet order (see from [line 45](https://github.com/alshedivat/al-folio/blob/master/_includes/header.html#L45)):
```HTML
<!-- Other pages -->
{% assign sorted_pages = site.pages | sort: "title" %}
{% for p in sorted_pages %}
{% if p.nav and p.autogen == nil %}
<li class="nav-item {% if page.title == p.title %}active{% endif %}">
  <a class="nav-link" href="{{ p.url | relative_url }}">
    {{ p.title }}
    {% if page.title == p.title %}
    <span class="sr-only">(current)</span>
    {% endif %}
  </a>
</li>
{% endif %}
{% endfor %}
```
If you want to change the order of the sections, you should simply reorder these blocks of code how you prefer!

**Reply by zeyunlu** [reply]
This exactly answers my questions! Thank you so much!

**Reply by yongjunshin** [reply]
Where can I find the 'header.html'?
The link in the above answer is not valid.

**Reply by george-gca** [reply]
This was a very old answer. If you check the same directory you'll find a file with the same name, different extension `header.liquid`. That is the one. A lot of files were renamed `.liquid` in #2048.
